feflow/protocols/nonequilibrium_cycling.py

```diff
@@ -349,6 +309,8 @@ def _execute(self, ctx, *, protocol, state_a, state_b, mapping, **inputs):
             softcore_LJ_v2 = True
         elif alchemical_settings.softcore_LJ.lower() == "beutler":
             softcore_LJ_v2 = False
+        # TODO: We need to test HTF for protein mutation cases, probably.
+        #  What are ways to quickly check an HTF is correct?
```

Comment:
Do you mean in terms of testing the hyrid system? Easiest way is to check if the endstates match the energy of the non-alchemical systems.